File: PharmacyUI/src/app/models/medicine.ts
export interface Medicine {
  id: number;
  name: string;
  notes: string;
  expiryDate: Date | string;
  quantity: number;
  price: number;
  brand: string;
}

export interface MedicineFormValue {
  Name: string;
  brand: string;
  notes: string;
  expiryDate: string;
  quantity: number;
  price: number;
}

File: Pharmacy/WebApplication1/Models/Medicine.cs
namespace WebApplication1.Models;
using System.Text.Json.Serialization;
public class Medicine
{
    public int Id { get; set; }

    [JsonPropertyName("name")]
    public string Name { get; set; } = string.Empty;

    public string Notes { get; set; } = string.Empty;

    [JsonPropertyName("expiryDate")]
    public DateTime ExpiryDate { get; set; }

    public int Quantity { get; set; }

    public decimal Price { get; set; }

    public string Brand { get; set; } = string.Empty;
}
